internal/k8sutil/k8sutil.go

```diff
@@ -94,6 +94,7 @@ func QuoteAttestationDeliver(
 	ctx context.Context,
 	c client.Client,
 	instanceName, namespace string,
+	requestType tcsapi.QuoteAttestationRequestType,
```

Comment:
Shouldn't this be `RequestTypeQuoteAttestation`?

Reply:
No, The argument type name is `QuoteAttestationRequestType`. The possible values are `RequestTypeQuoteAttestation` or `RequestTypeKeyProvisioning`. So, nothing wrong here !. Or did i miss something :(

Reply:
Ok, I get it now! I was confused with the string permutation for a minute. :-)